### ISSUE TITLE
chatgpt.com

### ISSUE THREAD

**Issue by panty-sniffer** [AdguardTeam/AdguardFilters#191873]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

Other ad blocker

### AdGuard version

N/A

### What type of problem have you encountered?

Missed analytics or tracker

### Which browser(s) do you use?

Firefox

### Which device do you use?

Desktop

### Where is the problem encountered?

https://chatgpt.com

### Ad Blocking

EasyList

### Add your comment and screenshots

The following URLs collect enormous amount of data continuously in background upto hundreds of network request per min.

```
||ab.chatgpt.com/v1/rgstr
||chatgpt.com/ces/v1/t
```

![image](https://github.com/user-attachments/assets/e562b892-7eea-4340-9357-8cbb35395c3b)

### Privacy

- [X] I agree to follow this condition

**Comment by Alex-302** [comment]
ab testing is not tracking.

**Comment by panty-sniffer** [comment]
@Alex-302  please check [easylist#19825](https://github.com/easylist/easylist/issues/19825) and [easyliist#19841](https://github.com/easylist/easylist/issues/19841)

TLDR: it is in fact tracking. but you can have both ab tested features by narrowing down what you block. I personally have chatgpt pro and despite blocking those domain I got ab tested features (check the second issue from a chatgpt employee).
